src/DocumentTemplate/tests/test_DT_Try.py

```diff
@@ -0,0 +1,221 @@
+import unittest
+
+from DocumentTemplate.DT_Util import ParseError
+from DocumentTemplate.DT_Raise import InvalidErrorTypeExpression
+
+
+class DT_Try_Tests(unittest.TestCase):
+    """Testing ..DT_Try.Try."""
+
+    def _get_doc_class(self):
+        from DocumentTemplate.DT_HTML import HTML
+        return HTML
+
+    doc_class = property(_get_doc_class,)
+
+    def test_DT_Try__Try____init__1(self):
+        """It allows only one else block."""
+        html = self.doc_class(
+            '<dtml-try>'
+            'Variable "name": '
+            '<dtml-except>'
+            '<dtml-else>'
+            '<dtml-var expr="str(100)">'
+            '<dtml-else>'
+            '<dtml-var expr="str(110)">'
+            '</dtml-try>')
+        self.assertRaisesRegexp(ParseError, '^No more than one else ', html)
+
+    def test_DT_Try__Try____init__2(self):
+        """It allows the else block only in last position."""
+        html = self.doc_class(
+            '<dtml-try>'
+            'Variable "name": '
+            '<dtml-else>'
+            '<dtml-var expr="str(100)">'
+            '<dtml-except>'
+            '</dtml-try>')
+        self.assertRaisesRegexp(
+            ParseError, '^The else block should be the last ', html)
+
+    def test_DT_Try__Try____init__3(self):
+        """It allows the else block only in last position."""
+        html = self.doc_class(
+            '<dtml-try>'
+            'Variable "name": '
+            '<dtml-except>'
+            '<dtml-finally>'
+            '<dtml-var expr="str(100)">'
+            '</dtml-try>')
+        self.assertRaisesRegexp(
+            ParseError, '^A try..finally combination cannot ', html)
+
+    def test_DT_Try__Try____init__4(self):
+        """It allows only one default exception handler."""
+        html = self.doc_class(
+            '<dtml-try>'
+            'Variable "name": '
+            '<dtml-except>'
+            '<dtml-var expr="str(100)">'
+            '<dtml-except>'
+            '<dtml-var expr="str(110)">'
+            '</dtml-try>')
+        self.assertRaisesRegexp(
+            ParseError, '^Only one default exception handler ', html)
+
+    def test_DT_Try__Try__01(self):
+        """It renders a try block if no exception occurs."""
+        html = self.doc_class(
+            '<dtml-try>'
+            'Variable "name": '
+            '<dtml-var expr="str(100)">'
+            '</dtml-try>')
+        res = html()
+        expected = ('Variable "name": 100')
```

Comment:
I think the parenthesis are not needed here and on some other places down the file.